app/src/main/java/com/omkarmoghe/pokemap/views/LoginActivity.java

```diff
@@ -254,5 +271,56 @@ public void onAnimationEnd(Animator animation) {
             mLoginFormView.setVisibility(show ? View.GONE : View.VISIBLE);
         }
     }
+
+    private void triggerAutoLogin() {
+
+        showProgress(true);
+
+        if (mPref.isUsernameSet() || mPref.isPasswordSet()) {
+            mNianticManager.login(mPref.getUsername(), mPref.getPassword());
```

Comment:
I would use the login method with the call back. As this method should be deprecated because with the callback method we can get more information on what went wrong. While with pokegoapi all we get is just a LoginFailed exception.